spec/integration/support/inspect_unmanaged_files_examples.rb

```diff
@@ -63,6 +63,12 @@ def parse_md5sums(output)
 
       expected_output = File.read("spec/data/unmanaged_files/#{base}")
       expected = expected_output.split("\n").select { |i| i.start_with?("  * ") }
+      unless actual.include?("  * /mnt/unmanaged/remote-dir/ (remote_dir)")
+        require 'byebug'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.